cobbler/utils.py

```diff
@@ -221,6 +221,46 @@ def is_systemd() -> bool:
     return False
 
 
+def is_supervisord() -> bool:
+    """
+    Return whether or not this system uses systemd.
+
+    This method currently checks if the path ``/usr/bin/supervisord`` exists.
+    """
+    if os.path.exists("/usr/bin/supervisord"):
+        return True
+    return False
```

Comment:
I believe this can be simplified.

```suggestion
    return os.path.exists("/usr/bin/supervisord"):
```

Reply:
Another (in my eyes preferred) option would be to use the RPC interface because we don't need to rely on exit codes or stdout parsing.

Docs: http://supervisord.org/api.html